### DISCUSSION TITLE
DOS 4.0 now under MIT license

### DISCUSSION THREAD

**Discussion by stuaxo** [dosemu2/dosemu2#2200] [General]
https://cloudblogs.microsoft.com/opensource/2024/04/25/open-sourcing-ms-dos-4-0/

So this isn't particularly useful, but it is interesting - as this was an early attempt at multi tasking.

I appreciate they released it under MIT and not some wonky license as well.

I'm shocked it's 10 years since they did the last release, hopefully it will be sooner that we get another - personally I want to see DOSSHELL.EXE QBASIC.EXE and EDIT.COM released (even though there are modern equivalents to all these).

**Comment by stsp** [comment]
https://github.com/microsoft/ms-dos
2.8K forks! This is popular. :)
Ok, waiting for 30 more years to see
win95 open-sourced.

**Comment by ecm-pushbx** [comment]
The git on our server (amd64 Debian Linux) somehow converted all the DOS line endings to Unix line endings on checkout. So the "nosrvbld" tool failed to find its data, erroring out of the build. I will see about fixing this (in a better way than running a mass unix2dos) to continue building.
